### ISSUE TITLE
Incomplete removal of components written in VRL-Studio

### ISSUE THREAD

**Issue by bsumirak** [VRL-Studio/VRL-Studio#15]
Define a new class via right-click on canvas: "New Groovy Code" and adding a **nested** class to it, like this:

    @ComponentInfo(name="MyClass", category="Custom")
    public class MyClass implements java.io.Serializable
    {
        private static final long serialVersionUID=1L;
        public class MyNestedClass{};
    }

Then compile it and right-click: delete it again in the component manager.

Take a look inside the VRL project file that you did this in: The "MyClass.class" file is correctly deleted, however, "MyClass$MyNestedClass.class" is not.

**Comment by miho** [comment]
Correct, we did this to prevent the necessary checks which delayed the deletion significantly. If you click on "development->build project" the class files of all deleted classes should be removed.
